src/debug/cmakeDebuggerTelemetry.ts

```diff
@@ -1,6 +1,7 @@
 import * as telemetry from "@cmt/telemetry";
 
 export const originatedFromLaunchConfiguration: string = "launchConfiguration";
```

Comment:
It would be nice to make this an Enum, similar to the `ConfigureTrigger` enum in cmakeproject.ts. This will make it easier to extend later on if we end up getting more specific or add more possibilities.